whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -17618,37 +17647,127 @@
             {"w": "11899145"}
           ]}},
           "result": {"mainEntity": {
+            "part": ["S.184-212"],
+            "isPartOf": [{
+              "@type": "Instance",
+              "describedBy": [{
+                "@type": "Record",
+                "controlNumber": "11899145"
+              }],
+              "hasTitle": [{
+                "@type": "Title",
+                "mainTitle": "Power of place : the religious landscape of the Southern Sacred Peak (Nanyue) in medieval China /"
+              }],
+              "identifiedBy": [{
+                "@type": "ISBN",
+                "value": "9780674033320 (cl : alk. paper)"
+              }],
+              "provisionActivityStatement": "cop. 2009",
+              "instanceOf": {
+                "@type": "Work",
+                "contribution": [{
+                  "@type": "PrimaryContribution",
+                  "agent": {
+                    "@type": "Agent",
+                    "label": "Robson, James,"
+                  }
+                }]
+              },
+              "marc:toDisplayNote": true
+            }]
+          }}
+        },
+        {
+          "name": "Example from https://www.kb.se/katalogisering/Katalogisering/artikelindexering/exempel/#14.",
```

Comment:
Good example. It's not obvious whether this should be part of a spec for 787 instead though, given that it spans multiple fields.

Reply:
I'm not sure it fits best with 773 or 787. That's an example of 'Bidragskatalogisering'. Normally (in marc21) this is described with 773 in combination with 787. I'm leaving this for now. We can move it when we gets to reviewOf examples.